src/plone/restapi/tests/test_documentation.py

```diff
@@ -837,6 +844,20 @@ def test_history_revert(self):
         response = self.api_session.patch(url, json={'version': 0})
         save_request_and_response_for_docs('history_revert', response)
 
+    @unittest.skipIf(not PLONE5, 'Just Plone 5 currently.')
```

Comment:
@jaroel how can we retrieve the controlpanel request/response if we just run the tests on Plone 4?